code/modules/background/space_sectors/coalition.dm

```diff
@@ -70,3 +70,494 @@
 	sector_welcome_message = 'sound/AI/welcome_weeping.ogg'//placeholder
 	sector_hud_menu = 'icons/misc/hudmenu/konyang_hud.dmi'
 	sector_hud_arrow = "menu_arrow"
+	lore_radio_stations = list("73.2 Navy Broadcasting Service", "122 Great Blue Dot", "75.4 PBA", "77.7 SoulFM", "78.1 RealFM")
+
+/datum/space_sector/haneunim/lore_radio_message(var/obj/item/lore_radio/R, var/chosen_station)
```

Comment:
This stuff really should be done differently, cause it makes simple sector definitions into huuuuge files with lots of noise. GitHub does not even want to show this file because "large diffs are not rendered by default".

Like the lore_radio_stations could be defined here sure, but the actual radio texts should be like a simple text file or something that is read by the radio. Like we have `config/names/` or whatever, idk if that's even used.

If the radio texts are just a simple text file, it would also make it easier for contributors to, well, contribute, if they don't have to write actual scary code with the whole += addtimer and type proc refs and whatever.

Reply:
geeves showed up and saved use from this nightmare